lib/tests/misc.nix

```diff
@@ -545,6 +546,12 @@ runTests {
         nonAbsolute = isStorePath (concatStrings (tail (stringToCharacters goodPath)));
         asPath = isStorePath (/. + goodPath);
         otherPath = isStorePath "/something/else";
+
+        caPath = isStorePath goodCAPath;
+        caPathAppendix = isStorePath
+          "${goodCAPath}/bin/python";
+        caAsPath = isStorePath (/. + goodCAPath);
```

Comment:
`/. + output` is generally not desirable, as it's usually not necessary, and requires IFD to do anything with it, but it's good to specify the behavior for this kind of value anyway.